rkt/prepare.go

```diff
@@ -48,7 +48,7 @@ func init() {
 	cmdRkt.AddCommand(cmdPrepare)
 
 	addStage1ImageFlag(cmdPrepare.Flags())
-	cmdPrepare.Flags().Var(&flagPorts, "port", "ports to expose on the host (needs contained networking with NAT)")
+	cmdPrepare.Flags().Var(&flagPorts, "port", "ports to expose on the host (requires --net). Syntax: --port=NAME:PORTNUMBER")
```

Comment:
Is `PORTNUMBER` the port on the host? If so, it can be documented as "Syntax: --port=NAME:HOSTPORT" to avoid the confusion between the port on the host and the port in the pod.

Ditto for other "PORTNUMBER" in this PR.